src/clib/cool1d_multi_g.F

```diff
@@ -299,6 +510,13 @@ subroutine cool1d_multi_g(
                   nother = (HeI(i,j,k) + HeII(i,j,k) +
      &                 HeIII(i,j,k))/4._DKIND +
      &                 HI(i,j,k) + HII(i,j,k) + de(i,j,k)
+#ifdef CALCULATE_TGAS_SELF_CONSISTENTLY
```

Comment:
This is iterating the gamma/temperature calculation for the changes in gamma from the H2 fraction. In testing, it seems to function correctly, but makes a negligible difference. Maybe this is ok to stay as an ifdef?